### ISSUE TITLE
Color Contrast for Critical and Online Colors - Dark Theme

### ISSUE THREAD

**Issue by MaximKipot** [grafana/grafana#22802]
Good day,

unfortunately, the colors used for alerts within notification channels are not accessible, meaning the contrast to the background is not enough.

**Steps to reproduce**:
open a graph and create a notification channel
in the alert tab define the condition and a value above which the value is alerting
now in your graph you will be getting green and red hearts

**Actual Result**:
the red color here is having 3.22 color contrast ratio
<img width="286" alt="Screenshot 2020-03-16 at 11 32 59" src="https://user-images.githubusercontent.com/1902515/76748874-d3300780-677b-11ea-9906-1f17e55d95a2.png">
3.99 is for the green color
<img width="267" alt="Screenshot 2020-03-16 at 11 33 31" src="https://user-images.githubusercontent.com/1902515/76748887-d925e880-677b-11ea-9c2a-32a8d7a6d685.png">

**Expected Result**

<img width="259" alt="Screenshot 2020-03-16 at 12 02 08" src="https://user-images.githubusercontent.com/1902515/76750214-299e4580-677e-11ea-9752-313695cdd77a.png">

<img width="271" alt="Screenshot 2020-03-16 at 12 00 19" src="https://user-images.githubusercontent.com/1902515/76749977-bbf21980-677d-11ea-970c-9d2693ea6b1f.png">

So the values should be replaced with smth like:

Within _variables.dark.generated.scss Instead of:
$online: #299c46;
$critical: #e02f44;

Use these or similar:
$online: #2ca94c;
$critical: #e96d7b;

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-contrast)

**Environment**:
- Grafana version: 6.7.0-pre
- Data source type & version: TestData DB coming with Simple dummy streaming example
- User OS & Browser: MacOS Mojave, Mozilla and Chrome

**Comment by dprokop** [comment]
@jessover9000 do you have any suggestions about colors to use to improve the accessibility in this case?

**Comment by dprokop** [comment]
as of https://github.com/grafana/grafana/pull/22831#issuecomment-599980617 : We are not ready to update our color palette yet, it will require a bigger overhaul and testing or a separate theme.

In the meantime upvote: #18041
